### ISSUE TITLE
IE10+: fire pointerdown and pointerup on click

### ISSUE THREAD

**Issue by wkeese** [theintern/leadfoot#17]
Calling `.click()` on IE10 doesn't emit an mspointerdown event, and similarly calling `.click()` on IE11 doesn't emit a pointerdown event.

I assume this is a webdriver problem but leadfoot is supposed to work around that.   Perhaps it's already firing mousedown/mouseup, but it would be good if it fired the pointer events too, since that's what actually happens when the user clicks on IE10+.

**Comment by csnover** [comment]
Is this still an issue?

**Comment by wkeese** [comment]
Yup, unfortunately still happens, see https://saucelabs.com/tests/444563213e524053bb29cbe2a4e8d150.

**Comment by wkeese** [comment]
FYI, I confirmed that on SauceLabs, Leadfoot uses native events on IE11/Windows 7, but not on IE11/Windows 8.1 nor IE11/Windows 10.

As listed in #83 this is presumably due to the `brokenMouseEvents` environment test failing on Win 8.1 and Win10.
